src/helpers/LatexHelpers.ts

```diff
@@ -0,0 +1,67 @@
+import { CacheType, CommandInteraction, MessageAttachment } from "discord.js";
+import svg2img, { ResvgRenderOptions } from "svg2img";
+var mjAPI = require("mathjax-node");
+
+
+export const Santinize = (message: string): string => {
+    let res_commands = [
+        [`\text{`, `\\backslash text~{`],
+        ['$', `\\$`],
+        ['"', '\\"'],
+        ['\\union','\\cup']
+    ]
+
+    let cleanedMessage = message
+    res_commands.forEach(([searchValue, replaceValue]) => {
+        cleanedMessage = cleanedMessage.replace(searchValue, replaceValue)
+    })
+    return cleanedMessage
+
+}
+
+export const EquationRender = async (cleanedMessage: string, interaction: CommandInteraction<CacheType>) => {
+    mjAPI.config({
+        MathJax: {
+            SVG: {
+                // scale: 70,
```

Comment:
After doing research, I found out that MathJax handles the scale by placing a `<span style="font-size:...">` element around the <svg> element, so this won't work in our case.